luaa.c

```diff
@@ -67,51 +67,82 @@ extern const struct luaL_Reg awesome_root_lib[];
 extern const struct luaL_Reg awesome_mouse_methods[];
 extern const struct luaL_Reg awesome_mouse_meta[];
 
-/** A call into the lua code aborted with an error
+/** A call into the lua code aborted with an error.
+ * This signal is used in the example configuration, @{05-awesomerc.md},
+ * to let a notification box pop up.
+ * @param err Table with the error object, can be converted to a string with
+ * `tostring(err)`.
  * @signal debug::error
  */
 
-/** A deprecated lua function was called
+/** A deprecated lua function was called.
+ * @param hint String with a hint on what to use instead of the
+ * deprecated functionality.
  * @signal debug::deprecation
  */
 
-/** An invalid key was read from an object, for example `c.foo`
+/** An invalid key was read from an object.
+ * This can happen if `foo` in an `c.foo` access doesn't exist.
+ * @param unknown1 Class?
+ * @param unknown2 Key?
  * @signal debug::index::miss
  */
 
-/** An invalid key was written to an object, for example `c.foo = "bar"`
+/** An invalid key was written to an object.
+ * This can happen if `foo` in an `c.foo = "bar"` assignment doesn't exist.
```

Comment:
Better to use `does not` instead of `doesn't` for doc/formal text.
(also empty line - not repeating that anymore for now ;))